### DISCUSSION TITLE
Socket timeout when using S3

### DISCUSSION THREAD

**Discussion by Shizanit** [mastodon/mastodon#17179] [Q&A] [answered]
Is it possible to increase the request time to S3? If so, how?

![image](https://user-images.githubusercontent.com/65107731/147101613-b9b8686f-5d36-4eea-abf1-0cf24723d518.png)

**Comment by ClearlyClaire** [comment, marked as answer]
There are environment variables for that purpose, `S3_OPEN_TIMEOUT` and `S3_READ_TIMEOUT`. They are both expressed in seconds and default to 5 seconds.
